uploadNeuroDB/mass_pic.pl

```diff
@@ -1,5 +1,36 @@
 #!/usr/bin/perl
 
+=pod
+
+=head1 NAME
+
+mass_pic.pl -- Generates check pic for the LORIS database system
+
+
+=head1 SYNOPSIS
+
+perl mass_pic.pl C<[options]>
+
+Available options are:
+
+-profile   : name of the config file in ../dicom-archive/.loris_mri
+
+-mincFileID: integer, minimum FileID to operate on
```

Comment:
C<> for FileID? if so, same thing for one line below

Reply:
done